pypeit/spectrographs/slitmask.py

```diff
@@ -951,6 +951,8 @@ def load_keck_deimoslris(filename:str, instr:str):
     # .decode() assumes encoding is the default 'utf-8'
     objname = [item.strip().decode() if isinstance(item, bytes) else item.strip()
                for item in hdu['ObjectCat'].data['OBJECT']]
+    # check if each objname is an ascii str (if not BinTableHDU later in the reduction will fail)
+    objname = [name if name.isascii() else 'None' for name in objname]
```

Comment:
Instead of setting the object name to None, would it make sense to instead remove the non-ascii characters?  After a quick Google search, I think you can do the following:
```
objname = [name  if name.isascii() 
                 else name.encode('ascii', 'ignore').decode() 
                 for name in objname]
```

Reply:
nice!

Reply:
Thanks @kbwestfall !! Always super helpful. I tried for a little bit to do this, but I did not figure it out.
I changed the code with your suggestion.